modules/kubernetes/config_schema.json

```diff
@@ -21,61 +19,38 @@
     "client_certificate": {
       "type": "string"
     },
-    "cluster_ca_certificate": {
+    "client_ca_certificate": {
```

Comment:
In go struct this is mentioned as [`cluster_ca_certificate`](https://github.com/odpf/entropy/blob/1eebb42d20b4e1541ef45dca40b08630d6511a01/pkg/kube/config.go#L28)

Reply:
any specific reason we chose to rename this field?